### PR TITLE
added styling info and image (documentation)

### LINKED ISSUE #3165: Add guidelines around new red delete styling on menu
We have added the ability to add a new styling to a "Delete" menu item.
(ie: Red text)

Add section under "Variations" for red menu item styling. Mention that it should be used for "Delete" actions only that are very destructive. 

Design issue: https://github.com/patternfly/patternfly-design/issues/1168

Implementation: https://github.com/patternfly/patternfly/issues/5057

### PR DESCRIPTION
**Pull request by JustinXHale**
closes #3165

### CODE REVIEW

File: packages/v4/patternfly-docs/content/design-guidelines/components/form-control/form-control.md [hunk 1]
```diff
@@ -46,11 +46,15 @@ Use the read only state when an input will *never* be editable. This state diffe
 
 <img src="./img/greyedout.png" alt="Gray-out input" width="644px"/>
 
+#### Red delete text styling
+* Red delete text styling should only be used for very destructive actions.
+* A divider should be used for destructive actions.
+
+<img src="./img/red-delete-text.png" alt="red text down down menu" width="863”/>
```
Comment (mmenestr) [inline packages/v4/patternfly-docs/content/design-guidelines/components/form-control/form-control.md:53]: forgot the "px" at the end of the width. not sure if that makes a difference but worth a try

**Review by JustinXHale — COMMENTED** [review]
updated with px on width line 53

**Review by JustinXHale — COMMENTED** [review]
testing updates

File: packages/v4/patternfly-docs/content/design-guidelines/components/menu/menu.md [hunk 2]
```diff
@@ -90,6 +90,12 @@ Use a drilldown menu when data is structured into levels and includes a long lis
 
 If data is more complex and has more than 2 levels, use a drilldown menu with breadcrumbs instead. Breadcrumbs offer better navigation between different levels. The number of visited levels is shown in a grey badge with a dropdown menu showing the menu items between the first and last level. The badge allows you to go back to a specific visited level. If you need to see the whole structure of your data, use a [tree view](/components/tree-view) instead. 
 
+### Red text menu
```
Comment (JustinXHale) [inline packages/v4/patternfly-docs/content/design-guidelines/components/menu/menu.md:93]: Added red text menu section

Comment (mmenestr) [reply packages/v4/patternfly-docs/content/design-guidelines/components/menu/menu.md:93]: I would edit this section a little bit to have a bit more description, and to not just be bullet points.

Suggestion:
If you have destructive items in a dropdown menu, you can optionally use red text styling for that item. A divider should be used to separate the destructive menu items from the non-destructive ones.

**Review by mmenestr — CHANGES_REQUESTED** [review]

**Review by mmenestr — APPROVED** [review]
Looks great @JustinXHale, merging!

**Pull request merged**